### DISCUSSION TITLE
Volume control through MCU

### DISCUSSION THREAD

**Discussion by Vahen1981** [tttapa/Control-Surface#435] [General]
Hi, First thaks for this amazing library!! It is really easy to use for someone like me, who has almost none experience programing.
After I programing succesfully my first generic control surface, I starting to thing in use the MCU protocol, but I have problems... probably it has a very simple solution, but I don´t know how to solve it... I'm trying to use the MCU protocol in my DAW (cubase) but I can not set the faders and make it work like volume controls of the daw. As I said, I'm trying to do this selecting "Mackie Control" in the DAW. I tried this code:

```
#include <Control_Surface.h>  
 
HairlessMIDI_Interface midi;
 
CD74HC4051 mux1 = {
  A0,       
  {2, 3, 4} 
};
 
CCPotentiometer volumePotentiometers[] = {
  { mux1.pin(0), { MCU::VOLUME_1} },
  { mux1.pin(1), { MCU::VOLUME_2} },
  { mux1.pin(2), { MCU::VOLUME_3} },
  { mux1.pin(3), { MCU::VOLUME_4} },
  { mux1.pin(4), { MCU::VOLUME_5} },
  { mux1.pin(5), { MCU::VOLUME_6} },
  { mux1.pin(6), { MCU::VOLUME_7} },
  { mux1.pin(7), { MCU::VOLUME_8} },
};
 
void setup() {
  Control_Surface.begin();  
}

void loop() {
  Control_Surface.loop();  
}
```

could someone help me?

**Comment by tttapa** [comment]
The MCU protocol uses pitch bend for volume control, not control change, you'll have to use `PBPotentiometer` instead of `CCPotentiometer`.

**Reply by Vahen1981** [reply]
Ok, thank you! I did like that and now works perfectly. Thank you very much!
